lib/src/buttons/switch.dart

```diff
@@ -65,7 +65,7 @@ class Switch extends StatelessWidget {
       onChanged: onChanged,
       dragStartBehavior: dragStartBehavior,
       activeColor: activeColor ??
-          context.maybeStyle?.primaryColor ??
+          context.macosTheme.primaryColor ??
```

Comment:
This should be `context.macosTheme?.primaryColor` because we are not calling `debugCheckHasMacosTheme`

Reply:
Good catch

Reply:
Or add the check `debugCheckHasMacosTheme` on every `build` method at each widget

Reply:
Don't think it's necessary to call it in every widget's `build` method, doing it at the `MacosApp` level should be fine I think